### PR TITLE
Prometheus, Loki: Update run queries button

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:

- renames `Run query` to `Run queries`
- hides `Run queries` button in Explore
- updates documentation

### CODE REVIEW

File: docs/sources/datasources/loki.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ The new field with the link shown in log details:
 
 Loki query editor is separated into 3 distinct modes that you can switch between. See docs for each section below.
 
-At the top of the editor there is `Run query` button that will run the query and `Explain | Builder | Code` tabs to switch between the editor modes. If the query editor is in Builder mode there are additional elements explained in the Builder section.
+At the top of the editor there is `Run queries` button that will run the queries and `Explain | Builder | Code` tabs to switch between the editor modes. If the query editor is in Builder mode there are additional elements explained in the Builder section.
+
+> **Note:** Run queries button is not present in Explore. To run Loki queries, use Explore's `Run query` button.
```
Comment (brendamuir) [inline docs/sources/datasources/loki.md:68]:
```suggestion
At the top of the editor, select `Run queries` to run a query. Select `Explain | Builder | Code` tabs to switch between the editor modes. If the query editor is in Builder mode, there are additional elements explained in the Builder section.

> **Note:** In Explore, `Run queries` is called `Run query`. To run Loki queries, select `Run query`.
```

File: docs/sources/datasources/prometheus.md [hunk 2]
```diff
@@ -47,7 +47,9 @@ Prometheus query editor is separated into 3 distinct modes that you can switch b
 
 ![Editor toolbar](/static/img/docs/prometheus/header-8-5.png 'Editor toolbar')
 
-At the top of the editor there is `Run query` button that will run the query and `Explain | Builder | Code` tabs to switch between the editor modes. If the query editor is in Builder mode there are additional elements explained in the Builder section.
+At the top of the editor there is `Run queries` button that will run the queries and `Explain | Builder | Code` tabs to switch between the editor modes. If the query editor is in Builder mode there are additional elements explained in the Builder section.
+
+> **Note:** Run queries button is not present in Explore. To run Prometheus queries, use Explore's `Run query` button.
```
Comment (brendamuir) [inline docs/sources/datasources/prometheus.md:52]: As above

File: docs/sources/datasources/loki.md (hunk 1 above)
Comment (ivanahuckova) [reply docs/sources/datasources/loki.md:68]: @brendamuir the changes looks good, but this below is not correct. We don't have that button in query editor in Explore and instead we have 1 button at the top of Explore.

```
In Explore, `Run queries` is called `Run query`. To run Loki queries, select `Run query`.
```

This is editor in dashboard:

image


This is editor in explore. No Run queries button, but there is Run query button on top ox Explore.
image

**Review by aocenas — APPROVED** [review]

File: docs/sources/datasources/loki.md (hunk 1 above)
Comment (brendamuir) [reply docs/sources/datasources/loki.md:68]: **Note:** In Explore, to run Loki queries, select `Run query`. does that make more sense?

Comment (ivanahuckova) [reply docs/sources/datasources/loki.md:68]: Yes!

**Comment by yesoreyeram** [comment]
@ivanahuckova - Just curious to know, Do we need **Run query** button when in `explain` mode? It is there just for visual consistency?

**Comment by ivanahuckova** [comment]
@yesoreyeram I think that the goal is to have all modes Explain-Builder-Code similar/same and only remove things that don't make any sense in that query mode. What are your thoughts on this?

**Comment by yesoreyeram** [comment]
Run query button in "explain" mode was bit confusing to me initially and looks redundant to dashboard's refresh query. User may confuse with both of those. 🤔  Also in explain mode, user not going to change any query so they can simply use dashboard's refresh query instead.
 
Not a big deal though. Just an UX thing.

**Pull request merged**